package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "cloudflare-worker-test-harness",
+  "version": "0.0.1",
+  "main": "index.js",
+  "dependencies": {
+    "jest-fetch-mock": "^1.6.5",
```

Comment:
you could move this to the dev dependency...

Reply:
I _think_ all of these can go to `devDependencies`...

Reply:
I'll leave `whatwg-fetch` as is unless anyone else feels strongly about it.

Reply:
leave whatwg-fetch as a dependency :)

Reply:
Fixed in 3a94948